modin/pandas/test/utils.py

```diff
@@ -888,24 +892,29 @@ def execute_callable(fn, inplace=False, md_kwargs={}, pd_kwargs={}):
         except Exception as pd_e:
             if check_exception_type is None:
                 return None
-            with pytest.raises(Exception) as md_e:
+            try:
                 if inplace:
                     _ = fn(modin_df, **md_kwargs)
                     try_cast_to_pandas(modin_df)  # force materialization
                 else:
                     try_cast_to_pandas(
                         fn(modin_df, **md_kwargs)
                     )  # force materialization
-            if check_exception_type:
-                assert isinstance(
-                    md_e.value, type(pd_e)
-                ), "Got Modin Exception type {}, but pandas Exception type {} was expected".format(
-                    type(md_e.value), type(pd_e)
+            except Exception as md_e:
+                if check_exception_type:
+                    assert isinstance(
+                        md_e, type(pd_e)
+                    ), "Got Modin Exception type {}, but pandas Exception type {} was expected".format(
+                        type(md_e), type(pd_e)
```

Comment:
Previously, we used md_e.value. Why did we discard it?

Reply:
This is an implementation detail of how pytest catches exceptions.  Now we rely on python itself.